object-manipulation-1/src/to-object.js

```diff
@@ -0,0 +1,12 @@
+/* exported toObject */
+
+function toObject(keyValuePair) {
+  var Object = {};
```

Comment:
change this to a small `o` , capital variable names are only for special circumstances.